environments/skeleton/{{cookiecutter.environment}}/ansible.cfg

```diff
@@ -13,3 +13,7 @@ filter_plugins = ../../ansible/filter_plugins
 [ssh_connection]
 ssh_args = -o ControlMaster=auto -o ControlPath=~/.ssh/%r@%h-%p -o ControlPersist=240s -o PreferredAuthentications=publickey -o UserKnownHostsFile=/dev/null
 pipelining = True
+
+[inventory]
+# Fail when any inventory source cannot be parsed.
+any_unparsed_is_failed = True
```

Comment:
Dumb question. Looks reasonable, but how does this relate to OFED? Is it so we fail earlier in the build?

Reply:
It doesn't relate to OFED tbh, I added this to catch the case when you have encrypted secrets in your inventory but don't have ANSIBLE_VAULT_PASSWORD_FILE set. Otherwise, it would fail to parse the inventory and skip a bunch of roles. 

I could split this into a separate PR?

Reply:
I'm fine with it here - cheers for clarifying